### PR TITLE
Rephrasing README and adding comments

### PR DESCRIPTION
**Pull request by cmaureir**

### CODE REVIEW

File: workshops/0_installation_party/README.md [hunk 1]
```diff
@@ -17,54 +17,66 @@ Our goal is to get you set up with the base things you will need to participate
...
+We recommend using [Visual Studio Code (VSC)](https://code.visualstudio.com) because it is popular among professional programmers, offers multiple languages support and is free to use. [Read more](https://realpython.com/python-ides-code-editors-guide/).
+
+In case you prefer to install another IDE, it's totally fine, and at the end
+will depend of your taste.
 
 ## Installing Git
 
 Git is a version control system which allows us to track changes in computer files. Files saved in Git can be "rewound" to find previous versions, which can be very useful when debugging!
 
-First!
```
Comment (sleepypioneer) [inline workshops/0_installation_party/README.md:50]: the extra two spaces here ensure in markdown that the next part is on a new line, but it doesn't really matter.

File: workshops/0_installation_party/README.md [hunk 2]
```diff
@@ -17,54 +17,66 @@ Our goal is to get you set up with the base things you will need to participate
...
-Check if you already have python installed:
+First!
+Check if you already have python installed by writing the following in a terminal:
 
 ```sh
 python --version
 ```
 
-If you see the following output then you already have python installed and don’t need to do anything: 
+Linux and macOS might be familiar with terminals, if you are using windows you
```
Comment (sleepypioneer) [inline workshops/0_installation_party/README.md]: should this say "linux and macOS _users_ might...." or it means something else? In general great idea to add how to get the terminal up

File: workshops/0_installation_party/README.md [hunk 3]
```diff
@@ -77,12 +89,12 @@ Once you have all these things set up (please ask coaches if you are at a setup
 - Use GitHub to fork this repository
 - Clone your fork to local machine using Git
 
-```sh
-git clone <>
-```
-
+  ```sh
+  git clone git@github.com:YOUR_USERNAME/resources
```
Comment (sleepypioneer) [inline workshops/0_installation_party/README.md:93]: oh yeh lol forgot to add this :see_no_evil: thank you!

**Review by sleepypioneer — COMMENTED** [review]
Awesome thanks you! just one question about one of the first comments but really grateful for all the improvements :pray:

File: workshops/0_installation_party/README.md (hunk 2 above)
Comment (cmaureir) [reply workshops/0_installation_party/README.md]:
```suggestion
Linux and macOS users might be familiar with terminals, if you are using windows you
```

File: workshops/0_installation_party/README.md (hunk 1 above)
Comment (cmaureir) [reply workshops/0_installation_party/README.md:50]: sorry, automatically my editor removes all the trailing spaces, so this got affected.

**Review by cmaureir — COMMENTED** [review]
adding some changes from the comments.

**Review by sleepypioneer — APPROVED** [review]
:shipit: I will go ahead and merge this :)

**Pull request merged**